Live-Snap/Live-Snap/Controllers/FetchSnapcodeViewController.swift

```diff
@@ -158,16 +158,18 @@ class FetchSnapcodeViewController: UIViewController, UITextFieldDelegate {
             return nil
         }
         
-        if result.count < 3 {
+        if result.count < 3 || result.count > 15 {
             return nil
         }
 
         return result
     }
     
     @objc func nextButtonWasPressed() {
-        guard let username = usernameTextField.text else { return }
-        guard let sanitizedUsername = sanitizeSnapchatUsernameString(username: username) else { showInvalidSnapchatUsernameError(); return }
+        nextButton.isUserInteractionEnabled = false
+        
+        guard let username = usernameTextField.text else { nextButton.isUserInteractionEnabled = true; return }
```

Comment:
combine these two guard lets into one and dont make them one line when theres this much code. Usually only make it a one liner if its short